src/Behat/Testwork/Cli/Application.php

```diff
@@ -99,16 +100,22 @@ public function doRun(InputInterface $input, OutputInterface $output)
             }
         }
 
-        if (is_file($path = $input->getParameterOption(array('--config', '-c')))) {
-            $this->configurationLoader->setConfigurationFilePath($path);
+        // Ignore all other inputs if config-reference is set:
+        if ($input->hasParameterOption(array('--config-reference'))) {
+            $input = new ArrayInput(array('--config-reference' => true));
         }
 
-        $this->add($this->createCommand($input, $output));
+        // Check the requested config file exists
```

Comment:
You should not commit these comments - if you think it needs explanation then maybe these pieces of logic can be extracted into private methods (or collaborators)